### PR TITLE
Feature/dataflow labels

### PR DESCRIPTION
**Pull request by peel**
Adds support for dataflow labels #6 and fixes constantly failing referer parser specs #9 (can't build otherwise)

### CODE REVIEW

File: src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/config.scala [hunk 1]
```diff
@@ -72,9 +73,22 @@ object config {
...
       )
 
+    private[this] def parseLabels(input: String) = {
+      val labelDelimiter = ";"
+      val valueDelimiter = "="
+
+      input
+        .split(labelDelimiter)
+        .map(_.split(valueDelimiter))
+        .filter(_.length == 2)
```
Comment (benjben) [inline src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/config.scala]: It would be great to not silently discard an error in the configuration.

**Review by benjben — APPROVED** [review]

File: src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/config.scala (hunk 1 above)
Comment (dilyand) [reply src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/config.scala]: Agreed, but shouldn't this happen in `Enrich.main()` where we parse the config?

File: src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/config.scala [hunk 2]
```diff
@@ -72,9 +73,22 @@ object config {
         bad,
         args.optional("pii"),
         resolver,
-        args.optional("enrichments")
+        args.optional("enrichments"),
```
Comment (dilyand) [inline src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/config.scala:76]: Shouldn't these new fields also be added to `ParsedEnrichConfig` and validated in `Enrich`?

**Review by dilyand — APPROVED** [review]
👍🏻

**Pull request merged**